### ISSUE TITLE
Unable to import 'distutils.util'

### ISSUE THREAD

**Issue by crazyscientist** [PyCQA/pylint#2955]
There are quite a few related issues (e.g. #1691), but none mention this particular issue. Somehow `distutils.util` cannot be processed by pylint.

### Steps to reproduce
1. Import something from `distutils.util`
2. Run `pylint`

### Current behavior
Pylint produces two errors for the import statement:

```
# In the file
from distutils.util import strtobool
```

```
# In the output
************* Module [...].run_agents
[...]run_agents.py:33:0: E0611: No name 'util' in module 'distutils' (no-name-in-module)
[...]run_agents.py:33:0: E0401: Unable to import 'distutils.util' (import-error)
```

### Expected behavior
None of the above error messages

### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]
```

**Comment by crazyscientist** [comment]
Looks like this might be a duplicate of #73...

**Comment by Pierre-Sassoulas** [comment]
If it is, it has been fixed 21 days ago in https://github.com/PyCQA/astroid/pull/672 and might be available in the next release.

**Comment by crazyscientist** [comment]
OK, just checked out the master branch and ran `pylint` on the problematic file.

Looks like the reason for the errors is the same as for #73, but it was not fixed by said pull request :(

**Comment by chingc** [comment]
I'm having the same issue with `import distutils.dir_util`

**Comment by PCManticore** [comment]
Thanks for confirming it's not working for you, we'll check why is that.

**Comment by AWhetter** [comment]
I can't reproduce this.
I'm using virtualenv 16.1.0 with

```
pylint 2.4.0-dev0
astroid 2.3.0
Python 3.6.8 (default, May 21 2019, 02:17:32) 
[GCC 8.3.0]
```

I have no custom config set on pylint and I'm running it on a file containing

```python
from distutils.util import strtobool
```

If you look at the fix in https://github.com/PyCQA/astroid/pull/672, is the condition that was added getting triggered? Are you able to find why not?

**Comment by crazyscientist** [comment]
> If you look at the fix in PyCQA/astroid#672, is the condition that was added getting triggered? Are you able to find why not?

Found it, actually using this unpublished version of astroid containing the fix helps :)
